ui/components/ui/callout/callout.js

```diff
@@ -41,7 +42,9 @@ export default function Callout({
         {children}
       </Typography>
       {dismiss && (
-        <i
+        <ButtonIcon
+          className="callout__close-button"
```

Comment:
Can we remove some CSS in favour of Box props? 

![Screenshot 2023-02-06 at 3 26 01 PM](https://user-images.githubusercontent.com/8112138/217110040-a76e2867-595f-4af0-a6eb-d2f9179c6568.png)